### ISSUE TITLE
scrape my library

### ISSUE THREAD

**Issue by sasagr** [leoncvlt/blinkist-scraper#27]
Would it be possible to scrape my all library using an argument like "library" and eventually skip what has been already scraped? This way I could setup a crono scraping and every time I have a new book in my library it will be downloaded.

**Comment by rocketinventor** [comment]
It is definitely possible. Also it is something that other people might
want, also.

There is a URL: "https://www.blinkist.com/api/books/library" that returns a
list of all the books in your library. And blinkist-scraper already skips
downloaded books, automatically. Seting up a chron task for this should
work okay.

However, if you have already run the program, you will probably find that
there are a lot of books there which you hadn't added yourself. - The
reason for this is that a book must be added to your library in order to
"read" (download) it.

**Comment by sasagr** [comment]
yes, I have run the program already many times and I m aware I need to to add the books I want to download to my library to be able to download them. What I want to avoid eventually it is to manually copy and paste the links to the download command.
Looking forward to see this new feature :)

**Comment by ebesser** [comment]
Did you guys figure out any solution for this that's straightforward?
